mod_reforged/hooks/entity/tactical/actor.nut

```diff
@@ -291,5 +318,35 @@
 					bro.addXP(XPgroup);
 			}
 		}
+
+		__original(_killer, _skill, _tile, _fatalityType);
 	}
 });
+
+::Reforged.QueueBucket.Late.push(function() {
+	::Reforged.HooksMod.hookTree("scripts/entity/tactical/actor", function(q) {
+		q.getLootForTile = @(__original) function( _killer, _loot )
+		{
+			// Change _killer to null to guarantee loot drop as vanilla checks for killer being null or belonging to Player or PlayerAnimals faction.
+			// Warning: this may be problematic for a mod that hooks this function and expects _killer to be the actual killer.
+			// Note: We do this in Late bucket so that mods that queue their hookTree after Reforged still receive the null _killer.
+			if (this.RF_canDropLootForPlayer(_killer))
+			{
+				return __original(null, _loot);
+			}
+
+			// Switcheroo getFaction to return None so that the generic vanilla faction check
+			// for loot fails i.e. _killer being of faction Player or PlayerAnimals.
+			local getFaction = _killer.getFaction;
```

Comment:
`_killer` might be `null` at this point.

We must find another dummy entity, to pass into `__original` with `none` as the faction, if we want to be able to turn off loot in every case

I recommend the dummy player